>>> 'Pkg.add("Gettext")' log
INFO: Installing Formatting v0.1.2
INFO: Installing Gettext v0.0.1
INFO: Installing PyCall v0.7.3
INFO: Package database updated

>>> 'using Gettext' log
Julia Version 0.4.0-dev+2839
Commit 8b4e9e9 (2015-01-21 02:39 UTC)
Platform Info:
  System: Linux (x86_64-unknown-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E5-2650 0 @ 2.00GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Sandybridge)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.3

>>> test log

WARNING: Set(a,b...) is deprecated, use Set(Any[a,b...]) instead.
 in depwarn at ./deprecated.jl:40
 in call at deprecated.jl:29
 in include at ./boot.jl:249
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:249
 in include_from_node1 at ./loading.jl:128
 in reload_path at ./loading.jl:152
 in _require at ./loading.jl:67
 in require at ./loading.jl:52
 in include at ./boot.jl:249
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:312
 in _start at ./client.jl:396
ERROR: LoadError: PyError (:PyObject_Call) <class 'UnicodeDecodeError'>
UnicodeDecodeError('ascii', b'Bonjour le monde20!', 16, 17, 'ordinal not in range(128)')
  File "/usr/lib/python3.4/gettext.py", line 498, in gettext
    return dgettext(_current_domain, message)
  File "/usr/lib/python3.4/gettext.py", line 462, in dgettext
    codeset=_localecodesets.get(domain))
  File "/usr/lib/python3.4/gettext.py", line 410, in translation
    t = _translations.setdefault(key, class_(fp))
  File "/usr/lib/python3.4/gettext.py", line 160, in __init__
    self._parse(fp)
  File "/usr/lib/python3.4/gettext.py", line 300, in _parse
    catalog[str(msg, charset)] = str(tmsg, charset)

 in include at ./boot.jl:249
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:312
 in _start at ./client.jl:396
while loading /home/idunning/pkgtest/.julia/v0.4/Gettext/test/runtests.jl, in expression starting on line 22

INFO: Testing Gettext
===============================[ ERROR: Gettext ]===============================

failed process: Process(`/home/idunning/julia04/usr/bin/julia --check-bounds=yes --code-coverage=none --color=no /home/idunning/pkgtest/.julia/v0.4/Gettext/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
INFO: No packages to install, update or remove
ERROR: Gettext had test errors
 in error at error.jl:19
 in test at pkg/entry.jl:717
 in anonymous at pkg/dir.jl:28
 in cd at ./file.jl:20
 in cd at pkg/dir.jl:28
 in test at pkg.jl:68
 in process_options at ./client.jl:234
 in _start at ./client.jl:396

>>> end of log
